File: src/main/webapp/gorod-drev-ui/src/app/shared/model/master.ts
export class Master {

  public constructor(private _id: number,
                     private _name: string,
                     private _surname: string,
                     private _phoneNumber: string) {

  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get surname(): string {
    return this._surname;
  }

  set surname(value: string) {
    this._surname = value;
  }

  get phoneNumber(): string {
    return this._phoneNumber;
  }

  set phoneNumber(value: string) {
    this._phoneNumber = value;
  }
}

File: src/main/java/cn/ua/goroddrev/domain/Master.java
package cn.ua.goroddrev.domain;

import cn.ua.goroddrev.domain.deserialize.MasterDeserializer;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;

import javax.persistence.*;
import java.io.Serializable;
import java.util.Set;

//@JsonDeserialize(using = MasterDeserializer.class)
@Entity
@Table(name = "masters")
public class Master implements Serializable {

    @Id
    @Column(name = "id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "name")
    private String name;

    @Column(name = "surname")
    private String surname;

    @Column(name = "phone_number")
    private String phoneNumber;

//    @JsonIgnore
//    @OneToMany(mappedBy = "master", fetch = FetchType.LAZY)
//    private Set<Product> products;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSurname() {
        return surname;
    }

    public void setSurname(String surname) {
        this.surname = surname;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

//    public Set<Product> getProducts() {
//        return products;
//    }

//    public void setProducts(Set<Product> products) {
//        this.products = products;
//    }
}
